2021-10-31 22:26:16,599 INFO    MainThread:6960 [wandb_setup.py:_flush():71] setting env: {}
2021-10-31 22:26:16,599 INFO    MainThread:6960 [wandb_setup.py:_flush():71] setting login settings: {}
2021-10-31 22:26:16,599 INFO    MainThread:6960 [wandb_init.py:_log_setup():357] Logging user logs to /home/dalab5/Projects/MA_packed/wandb/run-20211031_222616-1do83gzw/logs/debug.log
2021-10-31 22:26:16,600 INFO    MainThread:6960 [wandb_init.py:_log_setup():358] Logging internal logs to /home/dalab5/Projects/MA_packed/wandb/run-20211031_222616-1do83gzw/logs/debug-internal.log
2021-10-31 22:26:16,600 INFO    MainThread:6960 [wandb_init.py:init():390] calling init triggers
2021-10-31 22:26:16,600 INFO    MainThread:6960 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-10-31 22:26:16,600 INFO    MainThread:6960 [wandb_init.py:init():435] starting backend
2021-10-31 22:26:16,600 INFO    MainThread:6960 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-31 22:26:16,612 INFO    MainThread:6960 [backend.py:ensure_launched():199] starting backend process...
2021-10-31 22:26:16,616 INFO    MainThread:6960 [backend.py:ensure_launched():204] started backend process with pid: 6968
2021-10-31 22:26:16,617 INFO    MainThread:6960 [wandb_init.py:init():444] backend started and connected
2021-10-31 22:26:16,618 INFO    MainThread:6960 [wandb_init.py:init():503] updated telemetry
2021-10-31 22:26:16,619 INFO    MainThread:6960 [wandb_init.py:init():533] communicating current version
2021-10-31 22:26:17,233 INFO    MainThread:6960 [wandb_init.py:init():538] got version response 
2021-10-31 22:26:17,233 INFO    MainThread:6960 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2021-10-31 22:26:17,343 INFO    MainThread:6960 [wandb_init.py:init():576] starting run threads in backend
2021-10-31 22:26:19,216 WARNING MainThread:6960 [wandb_init.py:init():891] interrupted
Traceback (most recent call last):
  File "/home/dalab5/miniconda3/envs/GNN/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 870, in init
    run = wi.init()
  File "/home/dalab5/miniconda3/envs/GNN/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 581, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/home/dalab5/miniconda3/envs/GNN/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 216, in communicate_run_start
    result = self._communicate_run_start(run_start)
  File "/home/dalab5/miniconda3/envs/GNN/lib/python3.8/site-packages/wandb/sdk/interface/interface_queue.py", line 446, in _communicate_run_start
    result = self._communicate(rec)
  File "/home/dalab5/miniconda3/envs/GNN/lib/python3.8/site-packages/wandb/sdk/interface/interface_queue.py", line 232, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/dalab5/miniconda3/envs/GNN/lib/python3.8/site-packages/wandb/sdk/interface/router.py", line 32, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/dalab5/miniconda3/envs/GNN/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/home/dalab5/miniconda3/envs/GNN/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
